mapping-for-navigation/_posts/0000-01-09-modeling-interesections-for-map-navigation.md

```diff
@@ -0,0 +1,191 @@
+
+-   This place ☝️ [on the map](https://www.openstreetmap.org/edit#map=20.00/37.75213/-122.48172) 🗺
+
+Each way is a bidirectional road, the movement along which is possible in two directions simultaneously. The point at which the ways intersect denotes the intersection. Crossing ways without a common point make it impossible to build a route between the points that are at the ends of these ways.
+
+### 1a. Intersection and safety islands
+
+| ✅ **Recommended**  | ❌ **Not recommended** |
+|---|---|
+| [![][76c562c6]][76c562c6] | [![][787f1fd3]][787f1fd3] |
+| (_[fullsize][76c562c6]_ \| _[wireframe][b739583e] view_) | (_[fullsize][787f1fd3]_ \| _[wireframe][e0cba39d] view_) |
+
+  [76c562c6]: ../../images/intersection-modeling/simple_intersection_with_the_safety_islands.png "Simple intersection with the safety islands (fullsize view)"
+  [787f1fd3]: ../../images/intersection-modeling/simple_intersection_with_the_safety_islands_error.png "Incorrect drawing of the simple intersection with the safety islands (fullsize view)"
+  [b739583e]: ../../images/intersection-modeling/simple_intersection_with_the_safety_islands_wireframe.png "Simple intersection with the safety islands (wireframe view)"
+  [e0cba39d]: ../../images/intersection-modeling/simple_intersection_with_the_safety_islands_wireframe_error.png "Incorrect drawing of the simple intersection with the safety islands (wireframe view)"
+
+-   This place ☝️ [on the map](https://www.openstreetmap.org/edit#map=20.00/37.75405/-122.48078) 🗺
+
+In the presence of safety islands, located just before the crossroads, there is no need to divide the road onto dual carriageway. Set the nodes for safety islands and tag them as **`traffic_calming=island`**.
+
+### 2. Intersection of a bidirectional and dual carriageway roads.
+
+| ✅ **Recommended**  | ❌ **Not recommended** |
+|---|---|
+| [![][ac1e486c]][ac1e486c] | [![][14300d5a]][14300d5a] |
+| (_[fullsize][ac1e486c]_ \| _[wireframe][7064e664] view_) | (_[fullsize][14300d5a]_ \| _[wireframe][947d67cf] view_) |
+
+  [ac1e486c]: ../../images/intersection-modeling/bi-dir_dual_carriageway_intersection.png "Intersection of a bidirectional and dual carriageway roads (fullsize view)"
+  [7064e664]: ../../images/intersection-modeling/bi-dir_dual_carriageway_intersection_wireframe.png "Intersection of a bidirectional and dual carriageway roads (wireframe view)"
+  [14300d5a]: ../../images/intersection-modeling/bi-dir_dual_carriageway_intersection_error.png "Incorrect drawing of the intersection of a bidirectional and dual carriageway roads (fullsize view)"
+  [947d67cf]: ../../images/intersection-modeling/bi-dir_dual_carriageway_intersection_wireframe_error.png "Incorrect drawing of the intersection of a bidirectional and dual carriageway roads (wireframe view)"
+
+-   This place ☝️ [on the map](https://www.openstreetmap.org/edit#map=20/37.75984/-122.47683) 🗺
+
+No need to squeeze everything into one node bow-tie intersection to have sausage roads.
```

Comment:
I like the imagery of [bowtie pasta](https://en.wikipedia.org/wiki/Farfalle) with sausage. 😋

Reply:
👋 🍝 🌭  me too. Need to rewrite this sentence into more neutral style. 😊